src/App.js

```diff
@@ -1,11 +1,20 @@
 import React from 'react';
 import './App.css';
+import Homepage from "./pages/Homepage";
+
+
 
 function App() {
+
+
   return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
+
+
+
+    <>
+        <Homepage/>
+    </>
```

Comment:
De lege tags zijn overbodig, omdat het maar één child bevat
```suggestion
    <Homepage/>
```